testsuite/MDAnalysisTests/datafiles.py

```diff
@@ -473,5 +476,12 @@
 PDB_rama = resource_filename(__name__, 'data/19hc.pdb.gz')
 PDB_janin = resource_filename(__name__, 'data/1a28.pdb.gz')
 
+ITP_ala = resource_filename(__name__, 'data/gromacs_ala10.itp')
+XTC_ala = resource_filename(__name__, 'data/gromacs_ala10.xtc')
+
+ITP_atb = resource_filename(__name__, 'data/atb_hema.itp')
```

Comment:
Also these files are unused so either use them or lose them. You can also compress the files and MDA should just work still